.gitignore

```diff
@@ -2,4 +2,7 @@ node_modules/
 _esy/
 .merlin
 META
-.vscode/
\ No newline at end of file
+.vscode/
+.DS_Store
+*.tar.gz
+binaryen*
```

Comment:
Does this only hit files and not the directory?

Reply:
I'm pretty sure it only matches files at the root, but I can make it `/binaryen*/` if you want

Reply:
I think I was just confused. This is just to not include the opam file and any archives created manually?

Reply:
No, I think you actually caught an issue with this. While I was building the archives locally, I noticed that I left old untarred directories hanging around and I wanted to make sure they didn't end up in the other archives.

Reply:
I removed that line because it isn't really an issue with the new github action.